docs/collections/_policies/syntax-operators.md

```diff
@@ -562,63 +618,63 @@ when {
 
-**Usage:** `<long> - <long>`
+**Usage:** `<Long> - <Long>` or `- <Long>`
 
-As a binary operator with two operands, it subtracts the second long integer value from the first and returns a long integer difference.
+As a binary operator with two operands, it subtracts the second `Long` integer value from the first and returns a `Long` integer difference.
 
 #### Examples:
 {: .no_toc }
 
 ```cedar
-44 - 31                             // 13
-5 - (-3)                            // 8
--9,223,372,036,854,775,808 - 1 + 3  // overflow
+-3                              //-3
+44 - 31                         //13
+5 - (-3)                        //8
+-9223372036854775807 - 2 + 3    //error - overflow
+7 - "3"                         //error - second operand not a `Long`
 ```
 
-**Usage:** `- <long>`
-
-As a unary operator with one operand, it returns the negative of the value.
+Since the `-` symbol can mean both unary and binary subtraction, the third example must use parentheses to disambiguate.
 
-#### Examples:
-{: .no_toc }
-
-```cedar
--3
-```
+During policy validation, the validator will reject `-` expressions where either of the two operands is not a `Long`; so the last example is rejected by the validator. However, the validator does not attempt to detect possible integer overflow, so it does not reject the fourth example.
 
 ### `*` \(Numeric multiplication\) {#operator-multiply}
 
-**Usage:** `<long> * <long>`
+**Usage:** `<Long> * <Long>`
 
-Binary operator that multiplies two long integer values and returns a long integer product. One of the values ***must*** be an integer literal, the other value can be an integer literal or an expression that evaluates to an integer value.
+Binary operator that multiplies two `Long` integer operands and returns a `Long` integer product. One of the operands ***must*** be an literal, the other value can be a literal or a general expression or else the expression is [rejected by the parser](syntax-grammar.html#grammar-mult).
 
 {: .note }
 >There is no operator for arithmetic division.
 
 #### Examples:
 {: .no_toc }
 
+In these examples, suppose that `resource.value` is 3 and `context.budget` is 4.
+
 ```cedar
-10 * 20                          // 200
-resource.value * 10             // valid
-2 * context.budget > 100         // valid
-context.budget * context.limit   // not valid. One operand must be a constant
-9223372036854775807 * 2          // overflow
-5 * (-3)                         // -15
-5 * 0                            // 0
-"5" * 0                          // type error
+10 * 20                          //200
+resource.value * 10              //30
+2 * context.budget > 100         //false
+context.budget * resource.value  //will not parse - one operand must be a literal
+9223372036854775807 * 2          //error - overflow
+5 * (-3)                         //-15
+5 * 0                            //0
+"5" * 0                          //error - both operands must have type `Long`
```

Comment:
Following discussion on https://github.com/cedar-policy/rfcs/pull/50, we might add some examples using multiple literals. e.g., `2*3*resource.value` is ok, but `(2*3)*resource.value` is not ok because `(2*3)` is not a literal. I think `(-2)*resource.value` would also be rejected.

Reply:
These would fail to _parse_, correct? I.e., this is not a function of validation?

Reply:
correct. These fail to parse